README.md

```diff
@@ -1,2 +1,81 @@
 # AndroidStreamComparison
-Comparison of streaming libraries for Android with low latency
+Comparison of streaming libraries for Android with low latency.
+## Goal
+Give an overview of existing protocols for live-streaming and libraries for android.
+Design a system of two smartphones, one of which transmits live video from its camera and the other receives it.
+## Overview
+### Review protocols
+The following protocols for stream transmission over the network are considered:
+-	Protocols work on TCP:
+     -   RTMP
+     -   HTTP and related HLS, DASH и MPEG-DASH
+-	Protocols work on UDP, implementing tcp reliability at the application layer:
+     -   SRT
+     -   RTMFP
+-   Protocol works on SCTP:
+    -   WebRTC
+-  Protocol works on RTP:
```

Comment:
Maybe bntter to use the same level every where, because rtp also based on tcp or udp, but in this line level of abstraction is higher than in the previous line

Reply:
It is a transport protocol, so they are on the same level.

Reply:
But it uses udp..., so level isn't really same?